datalad_dataverse/__init__.py

```diff
@@ -15,13 +15,9 @@
         # specification of a command, any number of commands can be defined
         (
             # importable module that contains the command implementation
-            'datalad_dataverse.create_sibling_dataverse',
+            'datalad_dataverse.add_sibling_dataverse',
             # name of the command class implementation in above module
-            'CreateSiblingDataverse',
-            # optional name of the command in the cmdline API
-            'create-sibling-dataverse',
-            # optional name of the command in the Python API
-            'create_sibling_dataverse'
+            'AddSiblingDataverse',
```

Comment:
This needs the other entries, too, in order to build the manpages
```suggestion
            'AddSiblingDataverse',
            # optional name of the command in the cmdline API
            'add-sibling-dataverse',
            # optional name of the command in the Python API
            'add_sibling_dataverse'
```

Reply:
Thx! I edited the suggestion for clarity.